packages/ui/src/views/repo/pull-request/components/reviewers/pull-request-reviewers-header.tsx

```diff
@@ -28,78 +30,79 @@ const ReviewersHeader = ({
+    searchValue: searchQuery
+  })
+
+  const handleCloseValuesView = useRef(debounce(() => handleResetSearch(), 300)).current
 
   return (
-    <div className="flex items-center justify-between">
-      <span className="text-14 font-medium text-foreground-1">{t('views:pullRequests.reviewers')}</span>
-      <DropdownMenu.Root>
-        <DropdownMenu.Trigger className="group flex h-6 items-center px-2">
-          <Icon
-            className="text-icons-1 transition-colors duration-200 group-hover:text-icons-2"
-            name="vertical-ellipsis"
-            size={12}
-          />
-        </DropdownMenu.Trigger>
-        <DropdownMenu.Content className="w-[280px] p-0" align="end">
-          <div
-            className="relative flex items-center justify-between border-b border-borders-1 px-3 py-2.5"
-            role="presentation"
-            onKeyDown={e => e.stopPropagation()}
+    <div className="mb-0.5 flex items-center justify-between">
+      <h5 className="text-14 font-medium text-foreground-1">{t('views:pullRequests.reviewers', 'Reviewers')}</h5>
+
+      <DropdownMenu.Root onOpenChange={isOpen => !isOpen && handleCloseValuesView()}>
+        <DropdownMenu.Trigger asChild>
+          <Button
+            className="text-icons-1 hover:text-icons-2 data-[state=open]:text-icons-2"
+            size="xs_icon"
+            variant="custom"
           >
+            <Icon name="vertical-ellipsis" size={12} />
+          </Button>
+        </DropdownMenu.Trigger>
+        <DropdownMenu.Content className="w-80" align="end" sideOffset={-6} alignOffset={10}>
+          <div className="px-2 py-1.5" role="presentation" onKeyDown={e => e.stopPropagation()}>
             <SearchBox.Root
               className="w-full"
-              placeholder={t('views:pullRequests.searchUsers')}
-              value={searchQuery}
+              placeholder={t('views:pullRequests.searchUsers', 'Search users')}
+              value={search}
               handleChange={handleSearchChange}
               showOnFocus
             />
           </div>
+          <DropdownMenu.Separator />
 
-          <div className="p-1">
-            {usersList?.length === 0 && (
-              <div className="px-5 py-4 text-center">
-                <span className="text-14 leading-tight text-foreground-2">{t('views:pullRequests.searchUsers')}</span>
-              </div>
-            )}
-            <div className="max-h-[360px] overflow-y-auto">
-              {usersList?.length === 1 && usersList[0].uid === currentUserId ? (
-                <div className="px-5 py-4 text-center">
-                  <span className="text-14 leading-tight text-foreground-2">{t('views:pullRequests.noUsers')}</span>
-                </div>
-              ) : (
-                usersList?.map(({ display_name, id, uid }) => {
+          {!usersList?.length && (
+            <span className="block px-5 py-4 text-center leading-tight text-foreground-2">
```

Comment:
Instead of turning an inline element into a block, can we just use a block element?
```suggestion
            <div className="px-5 py-4 text-center leading-tight text-foreground-2">
```